lonboard/geopandas.py

```diff
@@ -0,0 +1,353 @@
+            constant scaler multiplied by elevation valuer, by default 1
+        alpha : float, optional
+            alpha (opacity) parameter in the range (0,1) passed to
+            mapclassify.util.get_color_array, by default 1
+        layer_kwargs : dict, optional
+            additional keyword arguments passed to lonboard.viz layer arguments (either
+            polygon_kwargs, scatterplot_kwargs, or path_kwargs, depending on input
+            geometry type), by default None
+        map_kwargs : dict, optional
+            additional keyword arguments passed to lonboard.viz map_kwargs, by default
+            None
+        classification_kwds : dict, optional
+            additional keyword arguments passed to `mapclassify.classify`, by default
+            None
+        nan_color : list-like, optional
+            color used to shade NaN observations formatted as an RGBA list, by
+            default [255, 255, 255, 255]. If no alpha channel is passed it is assumed to
+            be 255.
+        color : str or array-like, optional
+            single or array of colors passed to `lonboard.Layer` object (get_color if
+            input dataframe is linestring, or get_fill_color otherwise. By default None
+        wireframe : bool, optional
+            whether to use wireframe styling in deckgl, by default False
+        tiles : str or lonboard.basemap
+            either a known string {"CartoDB Positron", "CartoDB Positron No Label",
+            "CartoDB Darkmatter", "CartoDB Darkmatter No Label", "CartoDB Voyager",
+            "CartoDB Voyager No Label"} or a lonboard.basemap object, or a string to a
+            maplibre style basemap.
+        highlight: bool
+            whether to highlight each feature on mouseover (passed to
+            lonboard.Layer's auto_highlight)
+        m : lonboard.Map
+            an existing Map object to plot onto.
+
+        Returns
+        -------
+        lonboard.Map
+            a lonboard map with geodataframe included as a Layer object.
+        """
+        return _dexplore(
+            self._obj,
+            column,
+            cmap,
+            scheme,
+            k,
+            categorical,
+            elevation,
+            extruded,
+            elevation_scale,
+            alpha,
+            layer_kwargs,
+            map_kwargs,
+            classification_kwds,
+            nan_color,
+            color,
+            wireframe,
+            tiles,
+            highlight,
+            m,
+        )
+
+
+def _dexplore(
+    gdf,
+    column=None,
+    cmap=None,
+    scheme=None,
+    k=6,
+    categorical=False,
+    elevation=None,
+    extruded=False,
+    elevation_scale=1,
+    alpha=1,
+    layer_kwargs=None,
+    map_kwargs=None,
+    classification_kwds=None,
+    nan_color=[255, 255, 255, 255],
+    color=None,
+    wireframe=False,
+    tiles="CartoDB Darkmatter",
+    highlight=False,
+    m=None,
+):
+    """explore a dataframe using lonboard and deckgl
```

Comment:
If this docstring is exactly the same as the public-facing docstring, you can just write here to refer to the public docstring for parameter information.